client/src/containers/authenticated/Callback.js

```diff
@@ -0,0 +1,33 @@
+import React,{useEffect} from "react";
+import {createSession} from "../../reducers/actions/index"
+import loginGoogleProfile from "../../services/loginGoogle"
+import { useDispatch } from "react-redux";
+function Authenticate() {
+ const dispatch = useDispatch()
+useEffect(()=>{
+  getGoogleData()})
+
+  const getGoogleData =(e)=>{
+    loginGoogleProfile()
+    .then((res) => console.log(res)
+      /*   dispatch(
```

Comment:
I'm assuming you will be using this to complete authentication later on :)

If not, consider removing it from this file